### ISSUE TITLE
Accessibility Tab in Storybook builder is not working

### ISSUE THREAD

**Issue by Rajdeepc** [modernweb-dev/web#2683]
I am running my web component library with the below diagnostics.

```json

"dependencies": {
      "@storybook/addon-a11y": "^7.5.0",
      "@storybook/addon-essentials": "^7.5.0",
      "@storybook/addon-links": "^7.5.0",
      "@storybook/web-components": "^7.5.0",
      "@web/storybook-builder": "^0.1.7",
      "@web/storybook-framework-web-components": "^0.1.1",
      "storybook": "^7.5.0",
}
```
Whenever I go to my Accessibility Tab in my storybook I am getting the below error.
Need some help here!!

![Screenshot 2024-03-27 at 10 53 24 PM](https://github.com/modernweb-dev/web/assets/8790510/2022b9b8-9f57-4a07-93b8-204c85880a14)

This is my `main.js`

```js
const config = {
    stories: [
        '../packages/*/stories/*.stories.js',
        '../tools/*/stories/*.stories.js',
    ],
    addons: [
        '@storybook/addon-links',
        '@storybook/addon-essentials',
        '@storybook/addon-a11y',
    ],
    framework: {
        name: '@web/storybook-framework-web-components',
    },
    wdsFinal(config) {
        const json = fromRollup(rollupJson);
        return mergeConfigs(config, {
            nodeResolve: {
                exportConditions: ['browser', 'development'],
                moduleDirectories: [
                    'node_modules',
                    'packages',
                    'projects',
                    'tools',
                ],
            },
            clearTerminalOnReload: false,
            mimeTypes: {
                '**/*.json': 'js',
            },
            plugins: [json(), watchSWC()],
            http2: true,
            watch: true,
        });
    },
```

**Comment by bashmish** [comment]
@Rajdeepc I published a PR with what I had so far, but I'm running into a weird issue `RollupError: Identifier "_typeof" has already been declared` in the storybook static build generator, which I couldn't solve yet. Feel free to look into this if you want to help.

**Comment by bashmish** [comment]
Released in `@web/storybook-builder@0.1.10`
